astraguard/swarm/bus.py

```
@@ -0,0 +1,406 @@
+                )
+                self.metrics["lost"] += 1
+                return False
+        except Exception as e:
+            logger.error(f"ACK publish failed: {e}")
+            self.metrics["failed"] += 1
+            return False
+        finally:
+            self.pending_acks.pop(str(message.message_id), None)
+
+    async def _publish_reliable(
+        self, message: SwarmMessage, timeout_ms: int
```

Ruff (0.14.10)

Unused method argument: `timeout_ms`

(ARG002)